cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -83,10 +84,11 @@ func (i *deployCommand) Command() *cobra.Command {
 
 func (i *deployCommand) RegisterHook() {
 	i.command = &cobra.Command{
-		Use:     "deploy-service NAME SOURCE",
+		Use:     "deploy-service NAME [SOURCE]",
 		Short:   "Deploys a new Kogito Runtime Service into the given Project",
-		Aliases: []string{"deploy"},
-		Long: `deploy-service will create a new Kogito Runtime Service from source in the Project context.
+		Aliases: []string{"deploy", "create", "create-service"},
+		Long: `deploy-service will create a new Kogito Runtime Service in the Project context. 
+        If the source is provided, the build will take place on the cluster, otherwise you must upload the application binaries via "oc start-build [NAME-binary] --from-dir=target". 
 		Project context is the namespace (Kubernetes) or project (OpenShift) where the Service will be deployed.
 		To know what's your context, use "kogito use-project". To set a new Project in the context use "kogito use-project NAME".
```

Comment:
I'm not sure if I understood.

Reply:
Since some time, we have the `kogito project` command, just to display the current selected project.
The problem with `use-project` is that it also installs automatically the operator if not present.
To know the context, better to use `kogito project`. It does nothing but display which project is configured.